dwave/optimization/src/array.cpp

```diff
@@ -192,9 +191,7 @@ bool array_shape_equal(const Array* lhs_ptr, const Array* rhs_ptr) {
 
     return false;
 }
-bool array_shape_equal(const Array& lhs, const Array& rhs) {
-    return array_shape_equal(&lhs, &rhs);
-}
+bool array_shape_equal(const Array& lhs, const Array& rhs) { return array_shape_equal(&lhs, &rhs); }
```

Comment:
Please don't edit unrelated functions for formatting. It clutters the git history and without https://github.com/dwavesystems/dwave-optimization/issues/113 it's not even deterministic.